gitgud/levels/util.py

```diff
@@ -194,12 +192,40 @@ def test_level(level, test):
 
     return True
 
-
-class Level:
+class NamedList:
+    # Pass in list of tuples in format (reference, Object), where reference is of type str
+    def __init__(self, data=[]):
+        self.namedict = {d[0]:i for i, d in enumerate(data)}
```

Comment:
Rename to `self._name_dict`. First underscore indicates that the attribute is private, second underscore is convention and makes it easier to read